### PR TITLE
hw-mgmt: init: Fix hw-management init

### PR DESCRIPTION
**Pull request by sholeksandr**
Fix hw-management init issue. It took sometimes tool much time ~ 180sec

Bug:#3485095

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-ready.sh [hunk 1]
```diff
@@ -58,8 +58,15 @@ VMOD0014)
...
-		timeout 180 bash -c 'until [ -d /sys/devices/platform/mlxplat/mlxreg-hotplug/hwmon ]; do sleep 0.2; done'
+	arch=$(uname -m)
+	if [ "$arch" = "aarch64" ]; then
+		plat_path=/sys/devices/platform/MLNXBF49:00
+	else
+		plat_path=/sys/devices/platform/mlxplat
+	fi
+	if [ ! -d ${plat_path}/mlxreg-hotplug/hwmon ]; then
+		export plat_path
+		timeout 180 bash -c 'until [ -d ${plat_path}/mlxreg-hotplug/hwmon ]; do sleep 0.2; done'
```
Comment (ciju-nvidia) [inline usr/usr/bin/hw-management-ready.sh:69]: Could we use the 'retry_helper' function from hw-management-helpers.sh instead of this wait logic?

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-ready.sh (hunk 1 above)
Comment (sholeksandr) [reply usr/usr/bin/hw-management-ready.sh:69]: I's not a new code. It's backport from release branch.

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**